### PR TITLE
add button to filter questions by type

### PR DESCRIPTION
**Pull request by alexfauquette**
It add 4 buttons. During next loading user choice will be considered, except if the user unselect the 4 possiibilities.

image

### CODE REVIEW

File: src/Questions/index.js [hunk 1]
```diff
@@ -4,13 +4,40 @@ import axios from 'axios';
 import countries from './countries';
 import './questions.css';
 
+const toogleSelection = (listeOfObject, id) =>
```
Comment (teolemon) [inline src/Questions/index.js]: toogle >> toggle

Comment (alexfauquette) [reply src/Questions/index.js]: corrected ;)

**Review by ArnaudBarre — APPROVED** [review]
Thanks for adding this !

The code is good but I have some general feedback:
- There is a lot a custom CSS, I don't know foundation that is used for OFF but I think some classes of the framework could be reused
- I would have separate state from labels (with something like a const insightTypes = { id: questionLabel } (or [{id, question}] to be more extensible) and use 
`const [selectedInsights, setSelectedInsights] = useState(Object.keys(insightTypes))`

But as I said the code is good and I don't see any problem to merge this !

**Comment by alexfauquette** [comment]
Thanks for your feedback.

For the CSS, I don't know where I could found usefull CSS. Probably @teolemon  you have some resources links for that.

 Separate the state and the labels seems to be a good idea. I will do it tomorrow.

Thanks for this quick review :)

**Comment by ArnaudBarre** [comment]
I just merged another PR, so you should probably rebase to be sure there is no side effect with your PR :)

**Comment by alexfauquette** [comment]
I separated state from labels and put labels in a distinct file. Your right, it makes code more readable ;)

**Comment by ArnaudBarre** [comment]
👍 Good to put this in a separate file, this file is big enough 😅

**Pull request merged**